gridcomps/History/MAPL_EpochSwathMod.F90

```diff
@@ -29,27 +29,35 @@ module MAPL_EpochSwathMod
   use MAPL_DownbitMod
   use Plain_netCDF_Time
   use, intrinsic :: ISO_C_BINDING
-  use, intrinsic :: iso_fortran_env, only: REAL64
-  use ieee_arithmetic, only: isnan => ieee_is_nan
+  use MAPL_CommsMod, only : MAPL_Am_I_Root
   implicit none
-
   private
 
-  type, public :: samplerHQ 
+  integer, parameter       :: ngrid_max = 10
+
+  type, private :: K_V_CF
+     character(len=ESMF_MAXSTR) :: key
```

Comment:
A quick glance at the code suggests that an allocatable string would be better.   Maybe I missed a spot where it is passed to ESMF and needs to be a fixed-length string?

Otherwise, it would reduce the need for `trim()` later on.

Do not change for now - just something you should try to keep in mind when using string in Fortran.

Reply:
I will keep a note and change it in another PR.